salt/modules/jenkins.py

```diff
@@ -426,3 +426,31 @@ def get_job_config(name=None):
 
     job_info = server.get_job_config(name)
     return job_info
+
+
+def plugin_installed(name):
+    '''
+    .. versionadded:: Carbon
+
+    Return if the plugin is installed for the provided plugin name.
+
+    :param name: The name of the parameter to confirm installation.
+    :return: True if plugin exists, False if plugin does not exist.
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' jenkins.plugin_installed pluginName
+
+    '''
+
+    server = _connect()
+    plugins = server.get_plugins()
+
+    exists = filter(lambda x: name in x, plugins.keys())
```

Comment:
`exists = [x for x in plugins.keys() if name in x]` for compatibility across Python versions, please.